bin/rdygo-service.js

```diff
@@ -24,16 +22,9 @@ if (!process.argv.slice(2).length) {
   // parse json config file
   configuration = myConfig.init({
     path: path.resolve(program.configuration),
-    env: program.environment
+    env: program.environment,
   });
 
-  // start server using current configuration
-  createWorker(configuration, function(errWrker, server) {
-    if (errWrker) throw errWrker;
-    server
-      .listen(configuration.port, configuration.ip, function(err) {
-        if (err) throw err;
-        winston.info('Server listening on ', configuration.port, ' port');
-      });
-  });
+  // execute the cluster
+  cluster(configuration);
```

Comment:
Let's start with a simple node service now, instead of implementing the cluster ourselves, let's use ngnix or other proxy like solution